plugins/modules/make.py

```diff
@@ -49,6 +49,7 @@
   params:
     description:
       - Any extra parameters to pass to make.
+      - If the value is empty, only the key will be used. For example, V(FOO:) will produce V(FOO), not V(FOO=).
```

Comment:
I tried to document the change but do not know if this is clear enough.
I would appreciate a comment.

Reply:
Looks good to me, thanks!